tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -81,11 +81,27 @@ edpm_nodes:
         subnetName: subnet1
       - name: storagemgmt
         subnetName: subnet1
+edpm_nodes_networker:
+  standalone:
+    hostName: "{{ edpm_node_networker_hostname }}"
+    ansible:
+      ansibleHost: "{{ edpm_node_networker_ip }}"
```

Comment:
where is that coming from
in https://review.rdoproject.org/r/c/rdo-jobs/+/54113/ you have edpm_node_networker_ips but it isn't being set there, so probably it comes from the job definition?

but what about edpm_node_networker_ip ? it also is not being set in 54113